### DISCUSSION TITLE
Npm ERR! yallist not accessible from lru-cache

### DISCUSSION THREAD

**Discussion by fsacca88** [community/community#27042] [Actions]
<p>Hello everyone,<br>
hope you could please help with one of my Github Action’s job which is stucked from the 18th of May, when there have been issues with Actions all day long.<br>
The whole pipeline stops on the first automatic-test job,  as it reaches the “install” step, which execute an “npm ci” command (after a Node.js cache step), it produces the following error:</p>
<pre><code class="lang-auto">npm ERR! yallist not accessible from lru-cache

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-05-24T12_19_20_055Z-debug.log
</code></pre>
<p>This problem is up from the 18th and is still on after many re-runs.<br>
Fun fact: I have at least other 10 repos with the same identical pipeline and never had any problem like so.<br>
Is there any way to unstuck this situation?<br>
I attached the job file itself, thank you in advance.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/2/52fbc39ce3a32e10272b85a5be462a64f8d626fd.png" data-download-href="/uploads/short-url/bQ6yogFU0Vk2osrIvPsspOZziB7.png?dl=1" title="testStep" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/67392724/181101429-cb779cdc-1a69-491b-a46f-582f3e062bb1.png" alt="testStep" data-base62-sha1="bQ6yogFU0Vk2osrIvPsspOZziB7" width="487" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/2/52fbc39ce3a32e10272b85a5be462a64f8d626fd_2_487x500.png, https://aws1.discourse-cdn.com/github/original/3X/5/2/52fbc39ce3a32e10272b85a5be462a64f8d626fd.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/5/2/52fbc39ce3a32e10272b85a5be462a64f8d626fd.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/2/52fbc39ce3a32e10272b85a5be462a64f8d626fd_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">testStep</span><span class="informations">657×674 46.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by lirantal** [comment]
<p>Thanks for reporting it <a class="mention" href="/fsacca88">@fsacca88</a></p>
<p>I don’t see anything wrong by examining the workflow file.<br>
Is it still an issue? I wonder if it could’ve been either a networking issue, something on npm’s end, or perhaps a private package that is unavailable to download.</p>

**Comment by fsacca88** [comment, marked as answer]
<p>Thanks for replying, I had some builds to do in the last days and it seems the issue solved itself.<br>
I have no clue, thanks anyway.</p>
